### ISSUE TITLE
Cannot start vncserver on Ubuntu 20.04

### ISSUE THREAD

**Issue by fusentasticus** [VirtualGL/virtualgl#150]

**Comment by dcommander** [comment]
This belongs on the TurboVNC bug tracker, assuming it is related to TurboVNC.  You're going to need to provide more information regarding your environment and how you are trying to start the server.  Obviously it works for me, or else I wouldn't have claimed that it worked: https://turbovnc.org/Documentation/Compatibility22

**Comment by fusentasticus** [comment]
Thanks for reply which helped despite the empty message (not to be rude --- my finger slipped and I posted prematurely, sorry.) 

Now, I struggled quite a bit already with vglconnect/vglrun but the table by @gansgar https://github.com/VirtualGL/virtualgl/issues/129#issuecomment-678830029 was very helpful -- VGL_DISPLAY=:1 was the magic to get this functionality to work with a user logged onto console .  But my real goal is to run a vncserver on my Ubuntu 20.04 host, supported by the 3D acceleration of VirtualGL. I chose tigervnc server but that didn't go well --- see excerpt below line --- after I made sure that  `grep -i  wayland /etc/gdm3/custom.conf `  says `WaylandEnable=false`.

Anyways, I gather I should use Turbovnc instead :) So I will try that.

---

```
vncext:      VNC extension running!
vncext:      Listening for VNC connections on local interface(s), port 5902
vncext:      created VNC server for screen 0
X connection to :2 broken (explicit kill or server shutdown).
```
